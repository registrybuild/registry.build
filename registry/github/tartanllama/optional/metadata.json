{"id":105306008,"node_id":"MDEwOlJlcG9zaXRvcnkxMDUzMDYwMDg=","name":"optional","full_name":"TartanLlama/optional","private":false,"owner":{"login":"TartanLlama","id":1856614,"node_id":"MDQ6VXNlcjE4NTY2MTQ=","avatar_url":"https://avatars.githubusercontent.com/u/1856614?v=4","gravatar_id":"","url":"https://api.github.com/users/TartanLlama","html_url":"https://github.com/TartanLlama","followers_url":"https://api.github.com/users/TartanLlama/followers","following_url":"https://api.github.com/users/TartanLlama/following{/other_user}","gists_url":"https://api.github.com/users/TartanLlama/gists{/gist_id}","starred_url":"https://api.github.com/users/TartanLlama/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/TartanLlama/subscriptions","organizations_url":"https://api.github.com/users/TartanLlama/orgs","repos_url":"https://api.github.com/users/TartanLlama/repos","events_url":"https://api.github.com/users/TartanLlama/events{/privacy}","received_events_url":"https://api.github.com/users/TartanLlama/received_events","type":"User","user_view_type":"public","site_admin":false},"html_url":"https://github.com/TartanLlama/optional","description":"C++11/14/17 std::optional with functional-style extensions and reference support","fork":false,"url":"https://api.github.com/repos/TartanLlama/optional","forks_url":"https://api.github.com/repos/TartanLlama/optional/forks","keys_url":"https://api.github.com/repos/TartanLlama/optional/keys{/key_id}","collaborators_url":"https://api.github.com/repos/TartanLlama/optional/collaborators{/collaborator}","teams_url":"https://api.github.com/repos/TartanLlama/optional/teams","hooks_url":"https://api.github.com/repos/TartanLlama/optional/hooks","issue_events_url":"https://api.github.com/repos/TartanLlama/optional/issues/events{/number}","events_url":"https://api.github.com/repos/TartanLlama/optional/events","assignees_url":"https://api.github.com/repos/TartanLlama/optional/assignees{/user}","branches_url":"https://api.github.com/repos/TartanLlama/optional/branches{/branch}","tags_url":"https://api.github.com/repos/TartanLlama/optional/tags","blobs_url":"https://api.github.com/repos/TartanLlama/optional/git/blobs{/sha}","git_tags_url":"https://api.github.com/repos/TartanLlama/optional/git/tags{/sha}","git_refs_url":"https://api.github.com/repos/TartanLlama/optional/git/refs{/sha}","trees_url":"https://api.github.com/repos/TartanLlama/optional/git/trees{/sha}","statuses_url":"https://api.github.com/repos/TartanLlama/optional/statuses/{sha}","languages_url":"https://api.github.com/repos/TartanLlama/optional/languages","stargazers_url":"https://api.github.com/repos/TartanLlama/optional/stargazers","contributors_url":"https://api.github.com/repos/TartanLlama/optional/contributors","subscribers_url":"https://api.github.com/repos/TartanLlama/optional/subscribers","subscription_url":"https://api.github.com/repos/TartanLlama/optional/subscription","commits_url":"https://api.github.com/repos/TartanLlama/optional/commits{/sha}","git_commits_url":"https://api.github.com/repos/TartanLlama/optional/git/commits{/sha}","comments_url":"https://api.github.com/repos/TartanLlama/optional/comments{/number}","issue_comment_url":"https://api.github.com/repos/TartanLlama/optional/issues/comments{/number}","contents_url":"https://api.github.com/repos/TartanLlama/optional/contents/{+path}","compare_url":"https://api.github.com/repos/TartanLlama/optional/compare/{base}...{head}","merges_url":"https://api.github.com/repos/TartanLlama/optional/merges","archive_url":"https://api.github.com/repos/TartanLlama/optional/{archive_format}{/ref}","downloads_url":"https://api.github.com/repos/TartanLlama/optional/downloads","issues_url":"https://api.github.com/repos/TartanLlama/optional/issues{/number}","pulls_url":"https://api.github.com/repos/TartanLlama/optional/pulls{/number}","milestones_url":"https://api.github.com/repos/TartanLlama/optional/milestones{/number}","notifications_url":"https://api.github.com/repos/TartanLlama/optional/notifications{?since,all,participating}","labels_url":"https://api.github.com/repos/TartanLlama/optional/labels{/name}","releases_url":"https://api.github.com/repos/TartanLlama/optional/releases{/id}","deployments_url":"https://api.github.com/repos/TartanLlama/optional/deployments","created_at":"2017-09-29T18:44:22Z","updated_at":"2025-10-16T03:41:48Z","pushed_at":"2024-06-10T20:36:47Z","git_url":"git://github.com/TartanLlama/optional.git","ssh_url":"git@github.com:TartanLlama/optional.git","clone_url":"https://github.com/TartanLlama/optional.git","svn_url":"https://github.com/TartanLlama/optional","homepage":"https://tl.tartanllama.xyz","size":468,"stargazers_count":906,"watchers_count":906,"language":"C++","has_issues":true,"has_projects":true,"has_downloads":true,"has_wiki":true,"has_pages":false,"has_discussions":false,"forks_count":73,"mirror_url":null,"archived":false,"disabled":false,"open_issues_count":29,"license":{"key":"cc0-1.0","name":"Creative Commons Zero v1.0 Universal","spdx_id":"CC0-1.0","url":"https://api.github.com/licenses/cc0-1.0","node_id":"MDc6TGljZW5zZTY="},"allow_forking":true,"is_template":false,"web_commit_signoff_required":false,"topics":[],"visibility":"public","forks":73,"open_issues":29,"watchers":906,"default_branch":"master","permissions":{"admin":false,"maintain":false,"push":false,"triage":false,"pull":false},"temp_clone_token":"","network_count":73,"subscribers_count":26}