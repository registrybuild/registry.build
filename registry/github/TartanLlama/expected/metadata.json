{"id":108308364,"node_id":"MDEwOlJlcG9zaXRvcnkxMDgzMDgzNjQ=","name":"expected","full_name":"TartanLlama/expected","private":false,"owner":{"login":"TartanLlama","id":1856614,"node_id":"MDQ6VXNlcjE4NTY2MTQ=","avatar_url":"https://avatars.githubusercontent.com/u/1856614?v=4","gravatar_id":"","url":"https://api.github.com/users/TartanLlama","html_url":"https://github.com/TartanLlama","followers_url":"https://api.github.com/users/TartanLlama/followers","following_url":"https://api.github.com/users/TartanLlama/following{/other_user}","gists_url":"https://api.github.com/users/TartanLlama/gists{/gist_id}","starred_url":"https://api.github.com/users/TartanLlama/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/TartanLlama/subscriptions","organizations_url":"https://api.github.com/users/TartanLlama/orgs","repos_url":"https://api.github.com/users/TartanLlama/repos","events_url":"https://api.github.com/users/TartanLlama/events{/privacy}","received_events_url":"https://api.github.com/users/TartanLlama/received_events","type":"User","user_view_type":"public","site_admin":false},"html_url":"https://github.com/TartanLlama/expected","description":"C++11/14/17 std::expected with functional-style extensions ","fork":false,"url":"https://api.github.com/repos/TartanLlama/expected","forks_url":"https://api.github.com/repos/TartanLlama/expected/forks","keys_url":"https://api.github.com/repos/TartanLlama/expected/keys{/key_id}","collaborators_url":"https://api.github.com/repos/TartanLlama/expected/collaborators{/collaborator}","teams_url":"https://api.github.com/repos/TartanLlama/expected/teams","hooks_url":"https://api.github.com/repos/TartanLlama/expected/hooks","issue_events_url":"https://api.github.com/repos/TartanLlama/expected/issues/events{/number}","events_url":"https://api.github.com/repos/TartanLlama/expected/events","assignees_url":"https://api.github.com/repos/TartanLlama/expected/assignees{/user}","branches_url":"https://api.github.com/repos/TartanLlama/expected/branches{/branch}","tags_url":"https://api.github.com/repos/TartanLlama/expected/tags","blobs_url":"https://api.github.com/repos/TartanLlama/expected/git/blobs{/sha}","git_tags_url":"https://api.github.com/repos/TartanLlama/expected/git/tags{/sha}","git_refs_url":"https://api.github.com/repos/TartanLlama/expected/git/refs{/sha}","trees_url":"https://api.github.com/repos/TartanLlama/expected/git/trees{/sha}","statuses_url":"https://api.github.com/repos/TartanLlama/expected/statuses/{sha}","languages_url":"https://api.github.com/repos/TartanLlama/expected/languages","stargazers_url":"https://api.github.com/repos/TartanLlama/expected/stargazers","contributors_url":"https://api.github.com/repos/TartanLlama/expected/contributors","subscribers_url":"https://api.github.com/repos/TartanLlama/expected/subscribers","subscription_url":"https://api.github.com/repos/TartanLlama/expected/subscription","commits_url":"https://api.github.com/repos/TartanLlama/expected/commits{/sha}","git_commits_url":"https://api.github.com/repos/TartanLlama/expected/git/commits{/sha}","comments_url":"https://api.github.com/repos/TartanLlama/expected/comments{/number}","issue_comment_url":"https://api.github.com/repos/TartanLlama/expected/issues/comments{/number}","contents_url":"https://api.github.com/repos/TartanLlama/expected/contents/{+path}","compare_url":"https://api.github.com/repos/TartanLlama/expected/compare/{base}...{head}","merges_url":"https://api.github.com/repos/TartanLlama/expected/merges","archive_url":"https://api.github.com/repos/TartanLlama/expected/{archive_format}{/ref}","downloads_url":"https://api.github.com/repos/TartanLlama/expected/downloads","issues_url":"https://api.github.com/repos/TartanLlama/expected/issues{/number}","pulls_url":"https://api.github.com/repos/TartanLlama/expected/pulls{/number}","milestones_url":"https://api.github.com/repos/TartanLlama/expected/milestones{/number}","notifications_url":"https://api.github.com/repos/TartanLlama/expected/notifications{?since,all,participating}","labels_url":"https://api.github.com/repos/TartanLlama/expected/labels{/name}","releases_url":"https://api.github.com/repos/TartanLlama/expected/releases{/id}","deployments_url":"https://api.github.com/repos/TartanLlama/expected/deployments","created_at":"2017-10-25T18:19:53Z","updated_at":"2025-10-23T10:37:24Z","pushed_at":"2025-09-01T09:44:47Z","git_url":"git://github.com/TartanLlama/expected.git","ssh_url":"git@github.com:TartanLlama/expected.git","clone_url":"https://github.com/TartanLlama/expected.git","svn_url":"https://github.com/TartanLlama/expected","homepage":"https://tl.tartanllama.xyz","size":1962,"stargazers_count":1765,"watchers_count":1765,"language":"C++","has_issues":true,"has_projects":true,"has_downloads":true,"has_wiki":true,"has_pages":false,"has_discussions":false,"forks_count":156,"mirror_url":null,"archived":false,"disabled":false,"open_issues_count":64,"license":{"key":"cc0-1.0","name":"Creative Commons Zero v1.0 Universal","spdx_id":"CC0-1.0","url":"https://api.github.com/licenses/cc0-1.0","node_id":"MDc6TGljZW5zZTY="},"allow_forking":true,"is_template":false,"web_commit_signoff_required":false,"topics":[],"visibility":"public","forks":156,"open_issues":64,"watchers":1765,"default_branch":"master","permissions":{"admin":false,"maintain":false,"push":false,"triage":false,"pull":false},"temp_clone_token":"","network_count":156,"subscribers_count":34}